src/imaginghelper.js

```diff
@@ -689,8 +689,17 @@
      **/
     function onOpen() {
         this._haveImage = true;
-        this.imgWidth = this._viewer.viewport.contentSize.x;
-        this.imgHeight = this._viewer.viewport.contentSize.y;
+        var contentSizeViewport = {},
+            contentSize;
+        // check if world is defined
+        if (this._viewer.viewport.contentSize == undefined){
+          contentSizeViewport.x = this._viewer.world.getHomeBounds().width;
+          contentSizeViewport.y = this._viewer.world.getHomeBounds().height;
+          contentSize = this._viewer.world.getItemAt(0).viewportToImageCoordinates(contentSizeViewport.x, contentSizeViewport.y);
+        }
+        contentSize = this._viewer.viewport.contentSize || contentSize;
```

Comment:
This is redundant now that the `if` above is already testing `this._viewer.viewport.contentSize`. Maybe this should become an `else` clause instead.